docs/index.asciidoc

```diff
@@ -59,19 +59,26 @@ database by default.
 ==== Database Auto-update
 
 This plugin bundles Creative Commons (CC) license databases.
-Logstash checks for database updates every day. It downloads the latest and can replace the old database
+If the auto-update feature is enabled, Logstash checks for database updates every day. It downloads the latest and can replace the old database
```

Comment:
```suggestion
If the auto-update feature is enabled in `logstash.yml`(as it is by default), Logstash checks for database updates every day. It downloads the latest and can replace the old database
```

Reply:
This is a somewhat unusual situation in that we're talking about related settings that are in two different places (logstash.yml and plugin settings).  It would be good to indicate that the auto-update setting is something else and is enabled by default early on.  The rest of the section makes more sense after we set that initial context.

Otherwise, unless somebody reads the "Disable the auto-update feature" setting info, they won't have this piece of information.